logger/logWriter.go

```diff
@@ -34,7 +34,7 @@ type LogInfo struct {
 
 func (w Writer) Write(p []byte) (int, error) {
 	logEntry := string(p)
-	logEntries := strings.Split(logEntry, "\n")
+	logEntries := strings.Split(strings.ReplaceAll(logEntry, "\r\n", "\n"), "\n")
```

Comment:
Can't we use https://golang.org/pkg/bufio/#ScanLines to handle this better than having so many string operations?

Reply:
makes sense. @BugDiver - would you like to try this instead?